test/1.4/syntax/T_multidecls.dml

```diff
@@ -0,0 +1,54 @@
+/*
+  © 2022 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+device test;
+import "testing.dml";
+
+template t {
+    saved   (int i, float j) = (1, 2);
+    session (int k, float l) = (3, 4);
+
+    session (int q, float r);
+}
+
+group b is t {
+    saved (int m, float n) = (5, 6);
+    session (int o, float p) = (7, 8);
+
+    session (int s, float t);
+}
+
+method wrapper() -> (bool, int) {
+    return (false, 4);
+}
+
+method test() throws {
+    cast(b, t).q = 9;
+    b.r = 10;
+    b.s = 11;
+    b.t = 12;
+    local (bool x, int y) = (false, 4);
```

Comment:
It struck me that an alternative syntax that we haven't discussed is `local (bool, int) (x, y);`. With the thinking that `(bool, int)` is a type (ok not yet but maybe later), and `(x, y)` deconstructs this tuple type.

I think the current syntax looks nicer at first glance, but the idea has some merit and warrants a discussion before dismissed.

Reply:
Discussed and dismissed.